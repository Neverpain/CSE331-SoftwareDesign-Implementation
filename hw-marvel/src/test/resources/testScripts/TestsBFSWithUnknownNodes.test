# Tests output of finding path on nodes that are not contained
# within the dataset, prints the node not contained

CreateGraph g

AddNode g n1

FindPath g n1 x

FindPath g x n1

FindPath g x y

FindPath g y x

FindPath g y y


