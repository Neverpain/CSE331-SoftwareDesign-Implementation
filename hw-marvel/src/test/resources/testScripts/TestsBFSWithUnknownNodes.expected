# Tests output of finding path on nodes that are not contained
# within the dataset, prints the node not contained

created graph g

added node n1 to g

unknown: x

unknown: x

unknown: x
unknown: y

unknown: y
unknown: x

unknown: y
unknown: y